amberdata/src/endpoint/marketCap.ts

```diff
@@ -0,0 +1,37 @@
+import { Requester, Validator } from '@chainlink/external-adapter'
+import { ExecuteWithConfig, Config } from '@chainlink/types'
+
+export const Name = 'marketcap'
```

Comment:
```suggestion
export const NAME = 
```
Also we should keep this consistent